commcare-support-library/src/main/java/org/commcare/commcaresupportlibrary/BiometricIdentifier.java

```diff
@@ -0,0 +1,7 @@
+package org.commcare.commcaresupportlibrary;
```

Comment:
can we make sure that all biometric classes are in `org.commcare.commcaresupportlibrary.identity` package.